docs/tutorials/nb_shap_feature_elimination.ipynb

```diff
@@ -22,15 +22,15 @@
     " \n",
     "The main advantages of using this routine are:\n",
     "\n",
-    "- It uses a tree-based model to detect the complex relations between features and the target.\n",
+    "- It uses a tree-based,linear model to detect the complex relations between features and the target.\n",
     "- It uses SHAP importance, which is one of the most reliable ways to estimate features importance. Unlike many other techniques, it works with missing values and categorical variables.\n",
     "- Supports the use of [GridSearchCV](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GridSearchCV.html) and [RandomizedSearchCV](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.RandomizedSearchCV.html) in order to optimize hyperparameters at each iteration. This way you can assess if the removal of a given feature reduces the predictive power, or simply requires additional tuning of the model.\n",
     "- You can also provide a list of features, that should not be eliminated. E.g incase of prior knowledge.\n",
     "\n",
     "The disadvantages are:\n",
     "\n",
     "- Removing lowest [SHAP](https://shap.readthedocs.io/en/latest/) importance feature does not always translate to choosing the feature with lowest impact on model's performance. Shap importance illustrates how strongly a given feature affects the output of the model, while disregarding correctness of this prediction.\n",
-    "- Currently, the functionality only supports tree-based binary classifiers, in the future the scope might be extended.\n",
+    "- Currently, the functionality only supports tree-based,linear binary classifiers, in the future the scope might be extended.\n",
```

Comment:
wide range of models, however, the best picks are tree-based or linear.

I would rephrase it.